tests/acceptance/features/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Artur Neumann <info@jankaritech.com>
```

Comment:
I think this name needs to be changed.

Reply:
no it does not, I have made the file. It just landed in the same PR as we gave up to split the commits